provider/terrafomo-aws/gen/Terrafomo/AWS/Resource04.hs

```diff
@@ -6593,7 +6500,7 @@ data LaunchTemplateResource s = LaunchTemplateResource'
     , _disableApiTermination :: TF.Attr s P.Bool
     -- ^ @disable_api_termination@ - (Optional)
     --
-    , _ebsOptimized :: TF.Attr s P.Bool
+    , _ebsOptimized :: TF.Attr s P.Text
```

Comment:
interesting

Reply:
Wonder what caused this? I don't think there were any overrides for it